codecov.yml

```diff
@@ -0,0 +1,70 @@
+# Setting coverage targets per flag
+coverage:
+  status:
+    project:
+      default:
+        target: auto #overall project/ repo coverage
+      mux-audio:
```

Comment:
non-blocking: Looks like this is the recommended way of handling monorepos, but if possible, we should find a way to more easily keep the packages in sync with e.g. our `/packages`